### ISSUE TITLE
Confusion about the consent data captured into AEP DataLake

### ISSUE THREAD

**Issue by raresvla** [AdobeDocs/experience-platform.en#92]
Issue in ./help/landing/governance-privacy-security/consent/sdk.md

In the [Making consent-change commands](https://experienceleague.adobe.com/docs/experience-platform/landing/governance-privacy-security/consent/sdk.html?lang=en#commands) section, there's a highlight that reads the following:

> The setConsent command only updates data directly in the Profile store, and does not send any data to the Data Lake.

This is partially true, as it refers to the fact that the Profile Update Event is processed by the Unified Profile Service, but it's not (yet) captured into the Data Lake. 
This is true for any kind of _incremental update on the profile_ (when the profile is not replaced, but just a few properties are updated — in this case “consent”).

More importantly, **the note is actually inaccurate** when thinking of the Experience Event that is automatically captured on each consent change command. This Experience Event has the role of tracking consent changes over time (whereas the Profile Update Event only changes the profile attributes), providing the ability to answer a critical question like "When did the user gave/revoked consent x, y or z)?"

--- 

To sum up, we need to update the wording of that note to capture the following:
* each consent update command generates a Profile Update event (to update profile attributes) and an Experience Event (to keep track of changes over time)
* currently, it's just the latter (the Experience Event) which is captured into the Data Lake

**Comment by vfalon** [comment]
Thank you for submitting this issue. We will review the documentation and make the necessary updates as quickly as possible. Tracking internally in PLAT-128913.

**Comment by vfalon** [comment]
The documentation has been updated. Thanks for reporting this, Rares! Closing this issue.
